diff -ruN nekbone-2.3.4/src/cg.f nekbone/src/cg.f
--- nekbone-2.3.4/src/cg.f	2013-12-19 06:47:52.000000000 -0800
+++ nekbone/src/cg.f	2020-08-04 02:29:00.743609719 -0700
@@ -144,7 +144,7 @@
 
 c$OMP END PARALLEL
 
-    6    format('cg:',i4,1p4e12.4)
+    6    format('cg:',i4,1p4e17.10)
 
       if (nid.eq.0) then
         write(6,6) 0,sqrt(rnorminit)
diff -ruN nekbone-2.3.4/src/test_f77.f nekbone/src/test_f77.f
--- nekbone-2.3.4/src/test_f77.f	1969-12-31 16:00:00.000000000 -0800
+++ nekbone/src/test_f77.f	2020-08-04 00:23:57.530732801 -0700
@@ -0,0 +1,2 @@
+      subroutine test
+      end
diff -ruN nekbone-2.3.4/test/example1/makenek nekbone/test/example1/makenek
--- nekbone-2.3.4/test/example1/makenek	2013-12-19 22:13:49.000000000 -0800
+++ nekbone/test/example1/makenek	2020-08-04 22:38:06.015198335 -0700
@@ -6,10 +6,12 @@
 SOURCE_ROOT="../../src"
 
 # Fortran compiler
-F77="mpixlf77_r -qsmp=omp -qnosave"
+#F77="mpixlf77_r -qsmp=omp -qnosave"
+F77=smpiff
 
 # C compiler
-CC="mpixlc_r -qsmp=omp"
+# CC="mpixlc_r -qsmp=omp"
+CC=smpicc
 
 # pre-processor symbol list 
 # (set PPLIST=? to get a list of available symbols)
diff -ruN nekbone-2.3.4/test/example1/nekpmpi nekbone/test/example1/nekpmpi
--- nekbone-2.3.4/test/example1/nekpmpi	2013-12-17 14:08:11.000000000 -0800
+++ nekbone/test/example1/nekpmpi	2020-08-04 22:46:34.569586508 -0700
@@ -1,4 +1,7 @@
 rm -f logfile
 mv $1.log.$2 $1.log1.$2
-mpiexec -np $2 ./nekbone $1 > $1.log.$2
+smpirun -hostfile ../../../../topologies/hostfile-fattree-16.txt \
+	-platform ../../../../topologies/fattree-16.xml \
+	--cfg=smpi/host-speed:90000000000f \
+	-np $2 ./nekbone $1 > $1.log.$2
 ln $1.log.$2 logfile
